libraries/oneview_helper.rb

```diff
@@ -102,5 +102,44 @@ def convert_keys(info, conversion_method)
       end
       support
     end
+
+    # Get the diff of the current resource state and the desired state
+    # @param [OneviewSDK::Resource] resource Resource containing current state
+    # @param [Hash] desired_data Desired state for the resource
+    # @return [String] Diff string (multi-line)
```

Comment:
Yeah, I got it looking at the code... but it would be good to have in the comments if we are planning to generate documentation.